docs/how-to/debugging/vscode.rst

```diff
@@ -0,0 +1,88 @@
+================
+Debug via VSCode
+================
+
+Debugging is possible at different stages in your development process. It is
+different to debug a development app via ``briefcase dev`` than an bundled app
+that is build via ``briefcase build`` and run via ``briefcase run``.
+
+Development
+-----------
+During development on your host system you should use ``briefcase dev``. To
+attach VSCode debugger you can simply create a configuration like this,
+that runs ``briefcase dev`` for you and attaches a debugger.
+
+.. code-block:: JSON
+
+    {
+        "version": "0.2.0",
+        "configurations": [
+            {
+                "name": "Briefcase: Dev",
+                "type": "debugpy",
+                "request": "launch",
+                "module": "briefcase",
+                "args": [
+                    "dev",
+                ],
+                "justMyCode": false
+            },
+        ]
+    }
```

Comment:
I have no idea what to do with this block of JSON. We should either explain it here, or link to a VS Code documentation page.